src/Services/Authorizator.js

```diff
@@ -0,0 +1,16 @@
+import firebase from "firebase/app";
+import "firebase/auth";
+import { config } from "../Config/config";
+
+class Authorizator {
+
+    initializeApp = () => firebase.initializeApp(config);
+
+    authAsync = () => {
+        const provider = new firebase.auth.FacebookAuthProvider();
+        return firebase.auth().signInWithPopup(provider);
+    }
+
+    logOut = () => firebase.auth().signOut();
+}
+export const authorizatorClass = new Authorizator();
```

Comment:
authorizatorClass is strange. it's object, not class. and better to name it simply authorizator